.github/workflows/reusable_build_images.yml

```
@@ -215,14 +215,20 @@ jobs:
         with:
           path: artifacts
 
+      - name: Pull latest providers
+        run: cargo run pull
+        env:
+          GITHUB_TOKEN: ${{ secrets.PRIVATE_GITHUB_TOKEN }}
+
       - name: Create distributable zip file
         run: |
+          wasm_dir=$(cargo run config paths wasm-dir)
```

Not sure, but you might need the separator here
```suggestion
          wasm_dir=$(cargo run -- config paths wasm-dir)
```

---

Nah, that's only necessary if you want to pass options directly after `cargo run`. If you pass another command it understands you've moved on to the arguments of the command being run.